pkgs/development/python-modules/pymodes/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchPypi, buildPythonPackage, numpy, pyzmq }:
+
+buildPythonPackage rec {
+  pname = "pyModeS";
+  version = "2.11";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1r85scs99c87mqkjalw1wxzscnibyyvzdgyj6q8l0f8r5f2864c2";
+  };
+
+  propagatedBuildInputs = [ numpy pyzmq ];
+
+  # No tests included
+  doCheck = false;
+
+  pythonImportsCheck = [ "pyModeS" ];
+
+  meta = with lib; {
+    description = "Python Mode-S and ADS-B Decoder";
+    homepage = "https://github.com/junzis/pyModeS";
+    license = licenses.gpl3;
```

One file's header says `GPLv3+`.

```suggestion
    license = licenses.gpl3Plus;
```

---

Fixed.